realm/realm-jni/src/io_realm_internal_LinkView.cpp

```diff
@@ -176,3 +176,18 @@ JNIEXPORT jboolean JNICALL Java_io_realm_internal_LinkView_nativeIsAttached
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_LinkView_nativeFind
+  (JNIEnv *env, jobject, jlong nativeLinkViewPtr, jlong rowIndex)
```

Comment:
`rowIndex` -> `targetRowIndex` to match the name in core. `rowIndex` will be confusing in this context since it naturally means the index in the link view here.